src/Symfony/Component/Serializer/Normalizer/DateTimeNormalizer.php

```diff
@@ -164,4 +189,19 @@ private function getTimezone(array $context)
 
         return $dateTimeZone instanceof \DateTimeZone ? $dateTimeZone : new \DateTimeZone($dateTimeZone);
     }
+
+    private function isForceContextTimezone(array $context): bool
```

Comment:
`isContextTimezoneForced`?

Reply:
So you are answering my question about what name.  Your suggestion is
CONTEXT_TIMEZONE_FORCED

????